packages/build-essentials/src/webpack.config.js

```diff
@@ -33,8 +33,8 @@ const webpackConfig = {
                 loader: 'json-loader'
             },
             {
-                test: /\.(woff|woff2|eot|svg)$/,
-                loader: 'url-loader?limit=100000'
+                test: /\.(woff|woff2)$/,
```

Comment:
It was working like this before I implemented the typeface-work-sans npm module and it is working again, the loader was added because of errors during the build with this module, why should I add a dependency I don't need?